dataprofiler/profilers/profile_builder.py

```diff
@@ -2088,11 +2088,13 @@ def _update_row_statistics(
 
```

Comment:
if we want to do options and profiler, we need to also handle the `__add__` case. However, I suggest splitting up the PR.

@taylorfturner

Reply:
does line 211 check for that?

Reply:
its checkiing the first `ValueError` in `_add_error_checks` but 211 is not actually checking that the options are different because the test is setup by only changing the data in the profile and not setting up the profiles with different options.

Reply:
so add a new if statement in the `_add_error_checks` specifically for `options.row_statistics.null_count.is_enabled` or whatever the specific option call is for your new option